tasks/Linux.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Install Java
-  ansible.builtin.yum:
+  ansible.builtin.package:
```

Comment:
The package name changes with different package managers so this may not necessarily work across other flavours of linux. I like the idea though so can you make the name a varaible `{{ teamcity_package_java }}` and add this to the defaults file with the existing value so it doesn't break our existing setup.